index.js

```diff
@@ -11,4 +11,118 @@ function menubar() {
 }
 
 NavigationMenu.addEventListener('click', menubar);
-headerMenu.addEventListener('click', menubar);
\ No newline at end of file
+headerMenu.addEventListener('click', menubar);
+
+// form validation
+
+const form = document.querySelector('form');
+const clientName = document.getElementById('name');// get the name
+const email = document.getElementById('email');// get the email
+const messageType = document.querySelector('.errorMessage');// get the message
+const emailRegex = /^(([^<>()\\[\]\\.,;:\s@"]+(\.[^<>()\\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+const comment = document.getElementById('comment');
+let counter = 0;
+const ParagraphError = document.querySelector('.Paragraph-errors');
+
+// for setting errors for name
+const setError = (element, message) => {
+  messageType.textContent = message;
+  clientName.classList.add('error');
+  clientName.classList.remove('success');
+  ParagraphError.style.display = 'block';
+};
+
+// for setting errors for email
+const setErrorEmail = (element, message) => {
+  messageType.textContent = message;
+  email.classList.add('error');
+  email.classList.remove('success');
+  ParagraphError.style.display = 'block';
+};
+// for setting errors for comment
+const setErrorComment = (element, message) => {
+  messageType.textContent = message;
+  comment.classList.add('error');
+  comment.classList.remove('success');
+  ParagraphError.style.display = 'block';
+};
+
+// HERE I STARTED
+// HERE I STARTED
+// HERE I STARTED
+
+// for setting the the success message
+const setSuccess = () => {
+  messageType.textContent = '';
+  clientName.classList.add('success');
+  clientName.classList.remove('error');
+  email.classList.add('success');
+  email.classList.remove('error');
+  ParagraphError.style.display = 'block';
+};
+
+const validateInputs = () => {
+  const usernameValue = clientName.value.trim();
+  const emailValue = email.value.trim();
+  const messageValue = comment.value.trim();
+
+  // name validation
+  if (usernameValue === '') {
+    setError(clientName, 'Username is required');
+    return;
+  } if (usernameValue.length > 30) {
+    setError(clientName, 'Username should not exceed 30 characters or be less than 3');
+    return;
+  }
+  if (usernameValue.length < 3) {
+    setError(clientName, 'Username should not be less than 3');
+    return;
+  }
+  setSuccess(clientName);
+  counter += 1;
+
+  // email validation
+  if (emailValue === '') {
+    setErrorEmail(email, 'Email is required');
+    return;
+  }
+  if (emailValue !== emailValue.toLowerCase()) {
+    setErrorEmail(email, 'Email should be in lowercase');
+    return;
+  }
```

Comment:
- [ ] Good job with your email validation, it works as expected. But according to the project's requirements, the error message is supposed to show up close to the "submit" button i.e. **Get In Touch.** button. Kindly make sure you improve its styling and put it displaying on the **Get In Touch.** button and not at the top of the form.